go.mod

```diff
@@ -39,7 +39,7 @@ require (
 
 replace (
 	github.com/jteeuwen/go-bindata => github.com/jteeuwen/go-bindata v0.0.0-20151023091102-a0ff2567cfb7
-	github.com/openshift/api => github.com/openshift/api v3.9.1-0.20191028134408-abf28f93703b+incompatible
+	github.com/openshift/api => github.com/openshift/api v3.9.1-0.20191111211345-a27ff30ebf09+incompatible
```

Comment:
I've started seeing the pattern in some PRs of removing the replace line once the line in requires matches it (i.e. this line 42 and line 18)

should we jump on this pattern bandwagon ?